literature-cleaned.bib

```diff
@@ -17243,6 +17363,18 @@ @article{MF:TSE81
 	number={5},
 }
 
+@article{CI:AS81,
+	title= {{Rank Transformations as a Bridge Between Parametric and Nonparametric Statistics}},
+	author = {Conover, William J and Iman, Ronald L},
+	journal = {The American Statistician},

```

Comment:
s.a.

Reply:
Could not find an official abbreviation

Reply:
Then leave as is and probably add a comment that you could not find it, so nobody else starts searching for it again.